pkg/images/utils/utils.go

```diff
@@ -264,3 +264,26 @@ func FilterSuppressedCVEsNoClone(img *storage.Image) {
 		}
 	}
 }
+
+func IsRedHatImage(img *storage.Image) bool {
```

Comment:
Is this an exhaustive list of all potential Red Hat images (or perhaps we are explicitly only supporting these?). Related: will there be documentation for which images we will support?

Reply:
No need to document the logic. I just wanted to know if we will make it clear to users (and even ourselves) how we define what an "official Red Hat image" is. Sounds like that's your plan :)